api/controllers/case.js

```diff
@@ -93,25 +94,24 @@ router.get('/countsByCountry', function (req, res) {
  * @apiError NotAuthorized The user doesn't have permission to perform this operation.
  *
  */
-router.post('/new', function (req, res) {
+router.post('/new', function (req, res, next) {
   groups.user_has(req, 'Contributors', function () {
-    console.log("user doesn't have Contributors group membership")
+    log.debug("user doesn't have Contributors group membership")
```

Comment:
Another debug log, unless there's a reason to have both the log and the return message?